### ISSUE TITLE
Adding a custom board?

### ISSUE THREAD

**Issue by g7uvw** [amperka/ino#189]
Hi,

I'm trying to play with Ino using a Raspduino board [http://www.bitwizard.nl/wiki/index.php/Raspduino] on a Raspberry Pi.
The boards.txt entry for the Raspduino is:

```
raspduino.name=Raspduino
raspduino.upload.protocol=arduino
raspduino.upload.maximum_size=30720
raspduino.upload.speed=57600
raspduino.bootloader.low_fuses=0xff
raspduino.bootloader.high_fuses=0xda
raspduino.bootloader.extended_fuses=0x05
raspduino.bootloader.path=atmega
raspduino.bootloader.file=ATmegaBOOT_328_diecimila.hex
raspduino.bootloader.unlock_bits=0x3F
raspduino.bootloader.lock_bits=0x0F
raspduino.build.mcu=atmega328p
raspduino.build.f_cpu=16000000L
raspduino.build.core=arduino
raspduino.build.variant=standard
```

How do I communicate this information to Ino? I know it doesn't parse the boards.txt file, and I can't seem to specifiy most of these settings on the Ino command line.

Any ideas?

**Comment by g7uvw** [comment]
Ok, figured out how to do it, Documenting here for reference for others.

First off, I had to move the board entry further up the the boards.txt file, I added it as the second entry in the boards.txt file (not sure how far down the file is read, but ino list-models didn't list everything in the file)

Secondly, trashed the project directory (after backing up the code) and recreated it with ino init.

All seems to build fine now.

David
